app/src/main/java/edu/uc/zhukv/droneradarmap/ui/main/MainFragment.kt

```diff
@@ -30,9 +26,12 @@ class MainFragment : Fragment() {
         super.onActivityCreated(savedInstanceState)
         viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
         viewModel.airports.observe(viewLifecycleOwner, Observer{
-                airports -> actAirport.setAdapter(ArrayAdapter(context!!, R.layout.support_simple_spinner_dropdown_item, airports))
+                airports -> actAirport.setAdapter(ArrayAdapter(requireContext(), R.layout.support_simple_spinner_dropdown_item, airports))
         })
         viewModel.fetchAirports()
     }
 
+    companion object {
```

Comment:
Changed order of contents of class
https://kotlinlang.org/docs/coding-conventions.html#class-layout